main/FatesInterfaceMod.F90

```diff
@@ -466,6 +466,9 @@ module FatesInterfaceMod
       ! leaf maintenance respiration rate (umol CO2/m**2/s) 
       ! (NOT CURRENTLY USED, PLACE-HOLDER)
       !real(r8), allocatable :: lmrcanopy_pa(:)
+      
+      ! area weighted leaf water potential (MPa) 
+      real(r8), allocatable :: lwp_pa(:)
```

Comment:
Ryan,
  Thank you very much for pointting this point. I added this to the boundary layer so that I can link this to the leaf moisture for caluclation of transpiration in the host model. I can remove this change!